addons/web/static/src/webclient/user_group_ids/user_group_ids.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+<t t-name="web.ResUserGroups">
+    <div class="o_group row align-items-start">
```

Comment:
I would consider using OuterGroup and InnerGroup components here, to avoid replicating the produced DOM of a form view.

Also, I would write a unit test for this component.

I can do both myself if you want.